src/main/java/nl/tudelft/scrumbledore/GUI.java

```diff
@@ -423,10 +441,22 @@ public void handle(KeyEvent keyPressed) {
 
         // Mapping the desired keys to the desired actions.
         if (keyPress.equals("LEFT")) {
+          // Logging the movement of the player to the game log.
+          Logger.log("Player moved to " + player.getPosition().getX() + ", "
+              + player.getPosition().getY());
```

Comment:
Sure thing, where would you encourage I do movement tracking?  Directly from player,pos?